lib/src/core/fuzz/Cargo.toml

```diff
@@ -11,20 +11,12 @@ cargo-fuzz = true
 [dependencies]
 libfuzzer-sys = "0.4"
 bytes = "1.0.1"
-tokio = { version = "1", features = ["full"] }
+tokio = { version = "1.28", features = ["full"] }
```

Comment:
not sure if specifying a minor version of tokio is desired - I can remove this.

Reply:
Some users had problems with minor versions colliding with their own lock files so I took them off to make it easier for them. I don't think it matters on the fuzz file though